matrix_client/room.py

```diff
@@ -85,17 +88,14 @@ def display_name(self):
         # Member display names without me
         members = [u.get_display_name() for u in self.get_joined_members() if
```

Comment:
This should be `self.members_displaynames[u.user_id]` or the equivalent with appropriate default value (the way I read the spec anyway: https://matrix.org/speculator/spec/HEAD/client_server/unstable.html#calculating-the-display-name-for-a-room).

Reply:
Good catch. Although I think it should be `u.get_display_name(self)` if you agree with what I wrote below, as this method will take care of the default value.